.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -26,14 +26,24 @@ body:
   - type: input
     id: connect-version
     attributes:
-      label: Connect Version
+      label: Connect Version (if applicable)
       description: |
         Find this in Connect: System page → Info tab → under System Info.
         See https://docs.posit.co/connect/admin/server-management/#determine-version-installed
       placeholder: "e.g., 2025.07.0"
     validations:
       required: false
 
+  - type: input
+    id: connect-cloud-account-name
+    attributes:
+      label: Connect Cloud Account Name (if applicable)
+      description: |
+        Find this in Connect Cloud: Top right user menu.
+      placeholder: "e.g., My Account"
```

```suggestion
        Find this in Connect Cloud in the URL: connect.posit.cloud/my-account
      placeholder: "e.g., my-account"
```

https://connect.posit.cloud/mslynch95

If we're going to ask for the account, I think the slug name would be better.